pkg/kapis/devops/v1alpha2/register.go

```diff
@@ -621,8 +621,8 @@ func AddPipelineToWebService(webservice *restful.WebService, devopsClient devops
 			Produces("application/json", "charset=utf-8").
 			Doc("Convert jenkinsfile to json format. Usually the frontend uses json to show or edit pipeline").
 			Reads(devops.ReqJenkinsfile{}).
-			Returns(http.StatusOK, RespOK, devops.ResJson{}).
-			Writes(devops.ResJson{}))
+			Returns(http.StatusOK, RespOK, map[string]interface{}{}).
+			Writes(map[string]interface{}{}))
```

Comment:
Why changing to `map[string]interface`, this is not going to generate readable API documentation.

Reply:
As mentioned https://github.com/kubesphere/kubesphere/pull/3588#issue-600595695 here, we get the JSON data directly from Jenkins. It always causes problems if the Jenkins side makes changes. Users can read the document from Jenkins if they really needs that one.

Reply:
OK, please add these comments to codebase

Reply:
Done.